addons/point_of_sale/models/pos_session.py

```diff
@@ -258,6 +258,7 @@ def action_pos_session_closing_control(self):
             session.write({'state': 'closing_control', 'stop_at': fields.Datetime.now()})
             if not session.config_id.cash_control:
                 session.action_pos_session_close()
+        return True
```

Comment:
I don't the see the speed benefit of this particular line of diff

Reply:
Was just a way to call it in my shitty rpc script.

Reply:
your :poop: script fights like a :cow: